mailbox/store/src/main/java/org/apache/james/mailbox/store/search/MessageSearches.java

```diff
@@ -267,7 +267,8 @@ private boolean isInAttachments(String value, List<MessageAttachmentMetadata> at
             return textExtractor
                     .extractContent(
                         rawData,
-                        attachment.getType())
+                        // todo we likely want only the media type here
```

Comment:
fixed later in this PR?

Reply:
In a later commit ;-)

It's helpfull to explain my reviewers I know there is a problem, and yes that I will fix it.

Reply:
That could be a way to share that knowledge, we just have to all agree it's used for that purpose.